gradle.properties

```diff
@@ -16,7 +16,7 @@
 group=org.apache.kafka
 # NOTE: When you change this version number, you should also make sure to update
 # the version numbers in tests/kafkatest/__init__.py and kafka-merge-pr.py.
-version=0.11.0.0-SNAPSHOT
```

Comment:
I just noticed it via grepping and saw V_0_11_0_0 was added before it was actually released. In theory the LATEST_0_11_0 should be set to V_0_11_0_1, but it looks like those are just there for completeness currently and are not being used. LATEST_0_11 is used and the value of that should change but it looks like it wouldn't be affected by the change to the last digit. So maybe it's fine without the addition.